packages/design-mui/src/components/MuiFilters/MuiFilters.test.tsx

```diff
@@ -103,9 +129,304 @@ describe("components/MuiFilters", () => {
     })
 
     expect(setFilters).toHaveBeenCalledWith([
-      { field: "id", operator: "icontains", value: "was" },
+      { field: "name", operator: "icontains", value: "was" },
     ])
     expect(setPage).toHaveBeenCalledWith({ number: 1, size: 10 })
     vi.useRealTimers()
   })
+
+  it("Sets filter if value is empty and the operator is an empty type", async () => {
```

Comment:
Some of these tests are unrelated to my changes, but since I was adding tests here and in FilterRows, I went ahead and brought the coverage up to 100%.